src/Db/Service/CachedToggleStorage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Clearbooks\Labs\Db\Service;
+
+use Clearbooks\Labs\Db\Entity\Toggle;
+use Clearbooks\Labs\Toggle\UseCase\ToggleRetriever;
+
+class CachedToggleStorage implements ToggleRetriever
```

Comment:
Yes, but that will depend on the place of usage. I'm planning to use it in clearbooks/clearbooks and only instantiate it once there.